### ISSUE TITLE
Make available trough F-droid

### ISSUE THREAD

**Issue by forteller** [actorapp/actor-platform#55]
F-droid is the leading app store exclusively for Free and Open Source Software for Android. It would be the most natural thing in the world for Actor to be available there.

I want to use the app, but I don't want to connect my phone (my pocket computer that knows everything about me and a lot about my friends) to Google. Today that means that I can't use the app, because Google Play Store is the only place to get it.

Here is the information on how to submit an app to F-droid: https://f-droid.org/contribute/

I really hope you will be able to do this soon! Then you will also make Actor known among a lot of new people in the open source (development) community.

Thank you!

**Comment by ex3ndr** [comment]
Yes. We are planning to build Google-Free version. Actually in the past we had google-free version, but removed it to make mainaining a bit easier.

**Comment by iamale** [comment]
:+1:

**Comment by plato-cambrian** [comment]
Worth reading Moxie's arguments against doing so, 
https://github.com/WhisperSystems/TextSecure/issues/127

**Comment by pizzamaker** [comment]
+1 F-Droid!

**Comment by pizzamaker** [comment]
Submission queue thread: https://f-droid.org/forums/topic/actor/
